frontend/server/src/Controllers/User.php

```diff
@@ -4650,6 +4651,61 @@ public static function apiRevokeAPIToken(\OmegaUp\Request $r) {
         \OmegaUp\DAO\APITokens::deleteByName($r->user->user_id, $name);
         return ['status' => 'ok'];
     }
+
+    /**
+     * @return array{entrypoint: string, templateProperties: array{payload: VerificationParentalTokenDetailsPayload, title: \OmegaUp\TranslationString}}
+     *
+     * @omegaup-request-param string $parental_verification_token
+     */
+    public static function getVerificationParentalTokenDetailsForTypeScript(
+        \OmegaUp\Request $r
+    ): array {
+        $r->ensureIdentity();
+
+        $token = $r->ensureString(
+            'parental_verification_token',
+            validator: fn (string $token) => preg_match(
+                '/^[a-zA-Z0-9]{25}$/',
```

Comment:
```suggestion
                '/^[a-zA-Z0-9]{24}$/',
```
(I don't know why the generated token only contains 24 characters)